<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>netgame</class>
 <widget class="QMainWindow" name="netgame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>180</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>360</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>当前下棋方</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>当前状态</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>连接成功</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>我方所执棋</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/pic/bl.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开启联机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>当前棋面信息（邻接矩阵）</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
